src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -279,6 +284,37 @@ public function addClassResource(\ReflectionClass $class)
     }
 
     /**
+     * Checks whether the requested class, interface or trait exists and registers the result for resource tracking.
+     *
+     * @param string $class
+     *
+     * @return bool
+     *
+     * @final
+     */
+    public function getClassExists($class)
```

Comment:
It's not that it's not important : it's a side effect. A conditional one also. What matters most is that this returns if a class exists or not.